tests/handlers/aws/utils.py

```diff
@@ -36,18 +54,19 @@ def _load_file_fixture(name: str) -> str:
     return res
 
 
-def _class_based_id(klass: object, prefix: str = "", suffix: str = "") -> str:
+def _time_based_id(prefix: str = "", suffix: str = "") -> str:
```

Comment:
refactoring this to being a time based rather than class/method based, allows us to execute once across different methods (like the setup and the effective test ones) and just needing to care for keeping the state of the result where it must be, in order to have unique and not colliding ids across multiple invocations